Gemfile

```diff
@@ -40,6 +40,8 @@ gem 'dotenv-rails'
 gem 'exception_handler', '~> 0.8.0.0'
 # Use unsplash for nice pictures
 gem 'unsplash'
+# use activity_notification as notification system
+gem 'activity_notification', '~> 1.7'
```

Comment:
The reason is the 2.x series tiene breaking changes
https://github.com/simukappu/activity_notification/blob/master/CHANGELOG.md